CHANGELOG.md

```diff
@@ -4,13 +4,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
```

Comment:
This empty line and the one at the end don't do anything cause markdown requires 2 spaces. Please remove them to keep changes clean.